code/modules/client/preferences_savefile.dm

```diff
@@ -141,6 +141,8 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	if(current_version < 27)
 		if (!(underwear in GLOB.underwear_list))
 			underwear = "Nude"
+	if(current_version < 36)
+		eorg_arena = TRUE //Burn a version to force this on.
```

Comment:
This is pointless, save file bumping is for situations where the default value needs to be changed. The default value is already TRUE

Reply:
I ran it past crossed.